content/zh/docs/tasks/inject-data-application/define-environment-variable-container.md

```diff
@@ -29,34 +29,46 @@ content_template: templates/task
 {{< code file="envars.yaml" >}}
 
 1. 基于YAML文件创建一个Pod:
-
+       
+       ```shell
        kubectl create -f https://k8s.io/docs/tasks/inject-data-application/envars.yaml
+       ```
```

Comment:
These code blocks still not rendered quite right. You can fix it according to the English version.